test/test.js

```diff
@@ -20,11 +20,11 @@ describe("loader", function() {
 		}, "").should.be.eql(HEADER +
 			// First import
 			"var abc = (abc || {});\n" +
-			"abc.def = {};\n" +
+			"abc.def = abc.def || {};\n" +
```

Comment:
@michael-ciniawsky Should be two tests, the existing plus an additional test to cover the new functionality imo